deploy/backplane/srep/hypershift/service-cluster/10-backplane-dev.RoleBinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Remove this file.  I will put in a card and PR to remove this from hive with justification.